lib/parser.js

```diff
@@ -136,7 +136,54 @@ function createValuesObject(valuesFilePath) {
   return resultValues;
 }
 
+/*
+ * Export metadata as JSON.
+ */
+function generateMetadataObject(sections) {
```

Comment:
Please add some comments, at least about what this receives and what it returns, it will be really helpful in the future for other people.